cime_config/machines/config_machines.xml

```diff
@@ -60,7 +60,6 @@
 
   <machine MACH="miller">
     <DESC> ORNL AF cluster 800-node AMD Epyc 2-sockets 64-cores per node</DESC>
-    <NODENAME_REGEX>login.*</NODENAME_REGEX>
```

Comment:
@minxu74 this forces everyone on a node that starts with `login` to specify `--machine ` argument to create_[newcase,test]. Maybe it can be adjusted to a miller-specific string. Removing for now.